client/src/Components/Branding/Fonts/index.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 // also exported from '@storybook/react' if you can deal with breaking changes in 6.1
-import { Story, Meta } from '@storybook/react/types-6-0'
+import { Story, Meta } from 'react'
```

Comment:
This isn't imported from react, its a storybook component. Same with all other imports below